data/helpers.d/psql

```diff
@@ -143,7 +143,7 @@ ynh_psql_user_exists() {
 	# Manage arguments with getopts
 	ynh_handle_getopts_args "$@"
 
-	if [[ -n $(sudo --login --user=postgres PGUSER="postgres" PGPASSWORD="$(sudo cat $PSQL_ROOT_PWD_FILE)" psql -tAc "SELECT rolname FROM pg_roles WHERE rolname='$user';") ]]; then
+	if [[ -z $(sudo --login --user=postgres PGUSER="postgres" PGPASSWORD="$(sudo cat $PSQL_ROOT_PWD_FILE)" psql -tAc "SELECT rolname FROM pg_roles WHERE rolname='$user';") ]]; then
```

Comment:
I guess you know what you're doing but you sure about this right ? xD

Reply:
Same here, you would better try the return code I think.
And please don't use [[ ]] unless you really need them for their specific syntax.

Reply:
Same here: https://github.com/YunoHost/yunohost/blob/stretch-unstable/data/helpers.d/mysql#L145

Reply:
Indeed ... to me this also looks like a bug in the mysql helper as well ... (@maniackcrudelis wdu think ? ;D)

Reply:
Guess the hidden message is that I should have a look to the mysql helper ;)